src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/casenotes/CaseNoteMappingService.kt

```diff
@@ -171,6 +171,20 @@ class CaseNoteMappingService(
       .map { it.toDto() }
       .let { AllPrisonerCaseNoteMappingsDto(it) }
 
+  @Transactional
+  suspend fun updateMappingsByNomisId(oldOffenderNo: String, newOffenderNo: String) {
+    val count = repository.updateOffenderNo(oldOffenderNo, newOffenderNo)
+    telemetryClient.trackEvent(
+      "nonAssociation-mapping-merged",
```

Comment:
is this a non association?

Reply:
whoah nope ! Good spot 🤦

Reply:
will sort in next pr which is coming imminently